docs/messaging/topics.md

```diff
@@ -65,7 +65,46 @@ loopTopic.subscribe(async (ctx) => {
 });
 ```
 
-The limitation exists to avoid infinite loops in deployed functions, where a function calls itself indirectly via a topic. These sorts of mistakes can lead to large unintentional cloud charges - something you probably want to avoid.
+The limitation exists to avoid infinite loops in deployed functions where a function calls itself indirectly via a topic. These sorts of mistakes can lead to large unintentional cloud charges - something you probably want to avoid.
+
```

Comment:
Can we change this language to 'The limitiation exists to protect you from infinite loops'

Reply:
changed